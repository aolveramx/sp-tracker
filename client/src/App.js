import Dashboard from './components/Dashboard'
import Navbar from './components/layout/Navbar'
import Login from './components/Login'

import { DashboardContainer } from './components/styles/Dashboard/DashboardContainer.styled'
import GlobalStyles from './components/styles/layout/Global.styled'

const code = new URLSearchParams(window.location.search).get('code')

const App = () => {
  return (
    <div>
      <GlobalStyles />
      <DashboardContainer>
        <Navbar />
        {code ? <Dashboard code={code} /> : <Login />}
      </DashboardContainer>
    </div>
  )
}

export default App
