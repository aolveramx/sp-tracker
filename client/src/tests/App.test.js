import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import App from '../App'


describe('Tests in App.js', () => {

  test('should render <App />', () => {

    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()
    
  });
  
});