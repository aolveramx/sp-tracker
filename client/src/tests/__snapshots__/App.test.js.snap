// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in App.js should render <App /> 1`] = `
<div>
  <Memo(l) />
  <styled.div>
    <Navbar />
    <Login />
  </styled.div>
</div>
`;
